src/pages/graphql/schema/checkout/queries/get-payment-config.md

```diff
@@ -205,6 +224,26 @@ The following example runs the `getPaymentConfig` query for a `location: CHECKOU
                         "tagline": false
                     }
                 }
+                "google_pay": {
+                    "payment_source": "googlepay",
+                    "code": "payment_services_paypal_google_pay",
+                    "title": "Google Pay",
+                    "payment_intent": "authorize",
+                    "sort_order": "0",
+                    "sdk_params": [
+                        {
+                            "name": "src",
+                            "value": "https://www.paypal.com/sdk/js?client-id=..."
```

Comment:
Is this supposed to be a full URL or was it truncated?

Reply:
Truncated. You don't want to include sensitive data.